src/runtime/pkg/oci/utils_test.go

```diff
@@ -821,22 +821,22 @@ func TestRegexpContains(t *testing.T) {
 
 	//nolint: govet
 	type testData struct {
-		toMatch  string
```

Comment:
The reformatting of the test suite is welcome, but should be separate commit (I already [commented on this](https://github.com/kata-containers/kata-containers/pull/4492#pullrequestreview-1346095666)). I won't block this PR on this, since it's been open for a very long time. But if there is a respin, please consider it.

Reply:
Yeah, we have several changes in the CI and a dozen of rebases. Will update.

Reply:
You're looking at an outdated file...